src/main/java/uk/gov/hmcts/reform/bailcaseapi/domain/handlers/presubmit/UploadSignedDecisionNoticeHandler.java

```diff
@@ -71,19 +69,13 @@ public PreSubmitCallbackResponse<BailCase> handle(
         Document maybeSignedDecisionNotice = bailCase.read(UPLOAD_SIGNED_DECISION_NOTICE_DOCUMENT, Document.class)
             .orElseThrow(() -> new IllegalStateException("signedDecisionNotice is not present"));
 
-        Optional<List<IdValue<DocumentWithMetadata>>> maybeExistingTribunalDocuments =
```

Comment:
Can we keep the existing document collection? And just remove the unsigned document from the collection? 
In the future, there could be additional tribunal documents on the collection, and we don't want to remove all.

Reply:
I redone this work to remove the unsigned doc from the list instead of clearing the list.